src/promptflow-evals/promptflow/evals/evaluators/_groundedness/_groundedness.py

```diff
@@ -17,6 +17,9 @@
 
 
 class _AsyncGroundednessEvaluator:
+    PROMPTY_FILE = "groundedness.prompty"
+    LLM_CALL_TIMEOUT = 600
```

Comment:
Can this be changed to a single constant rather than redefined in each eval?

Reply:
Majorly for save/load consideration, each evaluator has its own folder.